j2cl-maven-plugin/src/it/translation/pom.xml

```diff
@@ -0,0 +1,16 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>translation-files</groupId>
+  <artifactId>parent</artifactId>
+  <version>1.0</version>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>pom_default.xml</module>
+    <module>pom_cz.xml</module>
+    <module>pom_fr.xml</module>
+  </modules>
```

Comment:
consider one more project with three <execution> blocks, one for each test, each with its own <configuration> block to specify the expected file, path

Reply:
This is what I'm talking about, more than one execution in the same pom file, with different xtb file and local specified per execution - so that the user doesnt have to copy the whole pom file, just the execution.